website/content/docs/agent/autoauth/methods/jwt.mdx

```diff
@@ -18,3 +18,13 @@ method](/docs/auth/jwt).
 - `remove_jwt_after_reading` `(bool: optional, defaults to true)` -
   This can be set to `false` to disable the default behavior of removing the
   JWT after it's been read.
+
+- `remove_jwt_follows_symlinks` `(bool: optional, defaults to false)` -
```

Comment:
I think this is ok.  That said, my intuition expected that with this and remove_jwt_after_reading set to true, both the symlink (and any intervening symlinks) as well as the ultimate JWT file referenced would be deleted.  An alternative to consider: leave remote_jwt_after_reading behaviour unchanged from prior to PR, i.e. it always only affects the filename referenced in the config, be it file or symlink, and make the new config option control whether or not the symlink target gets deleted.

Reply:
How about an alternative approach:
We rename `remove_jwt_follows_symlinks` to `remove_jwt_behavior`, a string with options `symlink` and `jwt` - it hopefully disambiguates the behaviour, as well as keeps the door open to support a `both`, in the future?

Reply:
I guess this could also be over-engineering the solution, too, as realistically I think the majority of customers won't want both (and the majority, in my intuition, will only want the `jwt` behaviour).

Reply:
Yeah, maybe we should just stick with what you've got, we've already spent a long time on what I believe to be a niche feature.

Reply:
Yeah - I agree that this is fairly niche. Thanks for the review, though, and thanks for sticking through with this!